pkg/api/alerting/v2alpha1/types.go

```diff
@@ -526,3 +526,10 @@ func NewBulkItemErrorServerResponse(ruleName string, err error) *BulkItemRespons
 		Error:     err,
 	}
 }
+
+// containsI reports whether substr is case-insensitive within s.
+func containsI(s, substr string) bool {
```

Comment:
`containsI` is not a readable function name, how about `containsCaseInsensitive`

Reply:
Do we really need a new function here?